File: experiments/arena/easc-csharp/EASCLibrary/RemoteException.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace Net.XpFramework.EASC
{
    public class RemoteException: IOException
    {
        #region Constructors
        public RemoteException(string message, Exception inner) : base(message, inner)
        {
        }
        #endregion
    }
}

File: ports/technologies/opt/easc/java/src/net/xp_framework/easc/remote/RemoteException.java
/* This class is part of the XP framework's EAS connectivity
 *
 * $Id$
 */

package net.xp_framework.easc.remote;

/**
 * Indicates an exception occured in the remote API.
 *
 */
public class RemoteException extends RuntimeException {

    public RemoteException(Throwable e) {
        super(e);
    }

    public RemoteException(String message, Throwable e) {
        super(message, e);
    }

    public RemoteException(String message) {
        super(message);
    }
}
